llama_stack/providers/utils/kvstore/postgres/postgres.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from datetime import datetime
+from typing import List, Optional
+
+import psycopg2
+from psycopg2.extras import DictCursor
+
+from ..api import KVStore
+from ..config import PostgresKVStoreConfig
+
+
+class PostgresKVStoreImpl(KVStore):
+    def __init__(self, config: PostgresKVStoreConfig):
+        self.config = config
+        self.table_name = "kvstore"
```

Comment:
should the config have a way to override the table name -- I think it should? because there might be a shared postgres instance people decide to point at?

Reply:
done.